src/modules/books-list.js

```diff
@@ -0,0 +1,63 @@
+const booksListEl = document.querySelector('.books-list');
+
+function uid() {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+}
+
+export default class Books {
+  books = [];
+
+  constructor() {
+    this.#fetchBooks();
+    this.#render();
+  }
+
+  add(book) {
+    book.id = uid();
+    this.books.push(book);
+    this.#persist();
+    this.#render();
+  }
+
+  remove(id) {
+    if (!id) return;
+
+    this.books = this.books.filter((book) => book.id !== id);
+    this.#persist();
+  }
```

Comment:
- [ ] Please kindly ensure that the book is actually removed from the localStorage. At the moment, it removes it from the DOM but when the page is refreshed it reappears.

Reply:
Hi @AnselemOdims, actually it's removing it from the localStorage, this.#persist() does that job.